contrib/ansible/roles/skydive_agent/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+galaxy_info:
+  author: Sylvain Baubeau
+  description:
+  company: Red Hat
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.0
```

Comment:
Since it use include_role, it should be bumped to 2.3

Reply:
Done